examples/jsm/loaders/LWOLoader.d.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  LoadingManager,
+  Material,
+  Object3D
+} from '../../../src/Three';
+
+export interface LWO {
+  materials: Material[];
+  meshes: Object3D[];
```

Comment:
When reading the code, I've realized that also instances of `THREE.Points` and `THREE.LineSegments` can be present in this array. Thus, I've used an array of `Object3D`s as the type.